python/py-htseq/Portfile

```diff
@@ -7,7 +7,7 @@ set _name           HTSeq
 set _n              [string index ${_name} 0]
 
 name                py-htseq
-version             0.11.2
+version             0.12.3
```

Comment:
this version requires Python 3.5+, so if you  want to keep the PY27 subport you'll need to pin that to an earlier version

Reply:
That's not correct: [version 0.12.3](https://github.com/htseq/htseq/tree/release_0.12.3) supports Python 2.7 and 3.5+ (see both README.md and setup.py).

Reply:
sorry, I glanced over it too quickly and misread their check for the Python version... you are totally right.